README.md

```diff
@@ -48,45 +48,47 @@ type Storage interface {
 
 ## 📑 Storage Implementations
 
-* [ArangoDB](/arangodb) <a href="https://github.com/gofiber/storage/actions?query=workflow%3A%22Tests+ArangoDB%22">
```

Comment:
![image](https://github.com/gofiber/storage/assets/7063188/1c056af6-854d-477e-9483-2ab8e4bb0521)
why did you changed the control character ?